Validate contexts when setting multiple contexts (#2022)
getsentry/sentry-ruby: CHANGELOG.md

- Do not report exceptions when a Rails runner exits with `exit 0` [#1988](https://github.com/getsentry/sentry-ruby/pull/1988)
- Ignore redis key if not UTF8 [#1997](https://github.com/getsentry/sentry-ruby/pull/1997)
  - Fixes [#1992](https://github.com/getsentry/sentry-ruby/issues/1992)

> - Validate that contexts set in set_contexts method are a Hash instances [#2022](https://github.com/getsentry/sentry-ruby/pull/2022/files)

Reviewer: grammar 

```suggestion
- Validate that contexts set in `set_contexts` are also Hash instances [#2022](https://github.com/getsentry/sentry-ruby/pull/2022/files)
```
